.github/workflows/build-site.yml

```diff
@@ -0,0 +1,39 @@
+name: Build & Deploy Site
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  deploy-to-gh-pages:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v1
+
+    - name: Use Node.js 12.x
+      uses: actions/setup-node@v1
+      with:
+        node-version: 12.x
+
+    - name: Install Packages
+      run: yarn install
+
+    - name: Build Site
+      run: cd site && yarn run build
+      env:
+        SPACE_ID: ${{ secrets.CONTENTFUL_SPACE_ID }}
+        ACCESS_TOKEN: ${{ secrets.CONTENTFUL_ACCESS_TOKEN }}
```

Comment:
I think the contentful token can be removed because we are no longer using contentful